korangar/src/graphics/passes/light_culling/shader/light_culling.wgsl

```diff
@@ -152,6 +165,20 @@ fn calculate_tile_vectors(tile_x: u32, tile_y: u32) -> mat4x3<f32> {
     return sides;
 }
 
+/// Creates a 3x3 rotation matrix that aligns the cone's axis with +Z.
+/// The cone's tip is assumed to be at the origin.
+fn create_cone_rotation_matrix(cone_axis: vec3<f32>) -> mat3x3<f32> {
+    let z = normalize(cone_axis);
+    let temp = select(vec3<f32>(1.0, 0.0, 0.0), vec3<f32>(0.0, 1.0, 0.0), abs(z.x) > 0.9);
+    let x = normalize(cross(temp, z));
+    let y = cross(z, x);
+    return mat3x3<f32>(
+        x.x, y.x, z.x,
+        x.y, y.y, z.y,
+        x.z, y.z, z.z
+    );
```

Comment:
I am not super happy with the naming here. Isn't what you called `x`, `y` and `z` here usually `row0`, `row1` and `row2` (or column, I don't matrix very often)?

Also rename `temp` please, at the very least it shoudn't be abbreviated.

Reply:
Added more context to make clear, what the vectors are meaning in this context (they are essentially the new x,y,z axis!).